<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportFinanceSummarySetupDialog</class>
 <widget class="QDialog" name="ReportFinanceSummarySetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Период</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="CDateEdit" name="edtBegDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblBegDate">
        <property name="text">
         <string>Дата начала периода</string>
        </property>
        <property name="buddy">
         <cstring>edtBegDate</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="CDateEdit" name="edtEndDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblEndDate">
        <property name="text">
         <string>Дата окончания периода</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbtnByFormingAccountDate">
        <property name="text">
         <string>формирования счета</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>103</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbtnByActionEndDate">
        <property name="text">
         <string>выполнения услуги</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lblEndDate</zorder>
     <zorder>lblBegDate</zorder>
     <zorder>edtBegDate</zorder>
     <zorder>horizontalSpacer</zorder>
     <zorder>edtEndDate</zorder>
     <zorder>rbtnByFormingAccountDate</zorder>
     <zorder>rbtnByActionEndDate</zorder>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStruct"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblOrgStruct">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblFinanceType">
     <property name="text">
      <string>Источник финансирования</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbFinanceType"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Типы обращений:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="CRBListBox" name="lstItems"/>
   </item>
   <item row="10" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblContract">
     <property name="text">
      <string>Договоры:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="CTableView" name="lstContracts">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>lblPerson</zorder>
  <zorder>label</zorder>
  <zorder>buttonBox</zorder>
  <zorder>lstItems</zorder>
  <zorder>lblFinanceType</zorder>
  <zorder>cmbPerson</zorder>
  <zorder>cmbFinanceType</zorder>
  <zorder>lblContract</zorder>
  <zorder>lblOrgStruct</zorder>
  <zorder>cmbOrgStruct</zorder>
  <zorder>lstContracts</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>CRBListBox</class>
   <extends>QListView</extends>
   <header>library/RBListBox.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportFinanceSummarySetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportFinanceSummarySetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
